### DISCUSSION TITLE
New Github pull request abuse?

### DISCUSSION THREAD

**Discussion by KNN-07** [community/community#83928] [Code Security] [answered]
![image](https://github.com/community/community/assets/55886589/7a77919b-f370-4c4c-aaaf-e2856b4ecf72)
Apparently the ping notifications is being abused to use github email address to send phishing emails? Or am I tripping?

Edit : the deleted ping link is here : https://github.com/RemiBaldy/spellCorrector/pull/1#issuecomment-1873473305

**Comment by yash-pal1** [comment]
this is trap

**Reply by KNN-07** [reply]
Yes i know, thats why i report it as phishing. But i just want to notify github about these type of scams 😭

**Comment by tunahancelik** [comment]
You should be careful, it looks like a trap

**Reply by Alokyadavv00** [reply]
Exactly!!

**Comment by karasowles** [comment, marked as answer]
Thanks for reporting this @KNN-07. 

@billythekid's response on a similar phishing attempt applies here as well:
https://github.com/orgs/community/discussions/81478#discussioncomment-8113627

**Comment by ShivamKumar2002** [comment]
Here's another example - https://github.com/ca98am79/connect-dynamodb/issues/21#issuecomment-1951789511

**Comment by webdevs0112** [comment]
It's possible that you're encountering a form of abuse or phishing attempt related to GitHub pull requests. While GitHub's notification system is typically used to inform users about pull requests, issues, or other activities related to their repositories, malicious actors may attempt to exploit these notifications for phishing purposes.
You should be careful.
